src/js/components/form-elements/v2/TextInput.jsx

```diff
@@ -25,7 +25,7 @@ const TextInput = ({
   >
     <input
       disabled={disabled}
-      className={`text-input ${errorMessage ? 'has-errors' : ''}`}
+      className={`form-element-input ${errorMessage ? 'has-errors' : ''}`}
       placeholder={placeholder}
       {...fieldProps}
```

Comment:
same here, be careful what you are overriding, the same example as I pointed out before applies here, if input will have errorMessages and className, the className prop will overwrite the 
```className={`form-element-input ${errorMessage ? 'has-errors' : ''}`}```